backend/src/database/state-migrations/12-rename-incorrect-images.ts

```diff
@@ -0,0 +1,51 @@
+import { StrictObject } from 'digital-fuesim-manv-shared';
+import type { Migration } from './migrations';
+
+export const renameIncorrectPatientImages12: Migration = {
+    actions: (_initialState, actions) => {
+        actions.forEach((action) => {
+            if (
+                (action as { type: string } | null)?.type ===
+                '[Patient] Add patient'
+            ) {
+                migratePatient(
+                    (action as { patient: { image: { url: string } } }).patient
+                );
+            }
+        });
+    },
+    state: (state) => {
+        (
+            state as {
+                patientCategories: {
+                    patientTemplates: { image: { url: string } }[];
+                }[];
+            }
+        ).patientCategories.forEach((patientCategory) =>
+            patientCategory.patientTemplates.forEach((patientTemplate) => {
+                if (patientTemplate.image.url === '/assets/male-patient.svg') {
+                    patientTemplate.image.url = '/assets/patient.svg';
+                }
```

Comment:
This code does essentially the same as `migratePatient()`/`migrateImageProperties()`.

Do you think it would make sense to use `migratePatient()` here, too? The most clean way might be to introduce a function `migratePatientTemplate()` that also calls `migrateImageProperties()` instead of using `migratePatient()` for patient templates, but since the typings in the migrations aren't that strict, one function for both should be fine IMO.

Reply:
This `migratePatient` function just seems to be not really necessary, couldn't you just call `migrateImapeProperties` directly (and doing the `patient.image` part in line 12)?

Reply:
> This `migratePatient` function just seems to be not really necessary, couldn't you just call `migrateImapeProperties` directly (and doing the `patient.image` part in line 12)?

Done